_lzbench/lzbench.cpp

```diff
@@ -233,6 +233,20 @@ size_t common(uint8_t *p1, uint8_t *p2)
     return size;
 }
 
+/*
+ * Allocate a buffer of size bytes using malloc (or equivalent call returning a buffer
+ * that can be passed to free). Touches each page so that the each page is actually
+ * physically allocated and mapped into the process.
+ */
+void *alloc_and_touch(size_t size, bool must_zero) {
```

Comment:
@Cyan4973 - I should also added that `memset` was my initial approach and that's how I discovered that, well, compilers are pretty smart.